nexus/src/app/vpc.rs

```diff
@@ -62,7 +70,7 @@ impl super::Nexus {
         )?;
         let (authz_vpc, db_vpc) = self
             .db_datastore
-            .project_create_vpc(opctx, &authz_project, vpc)
+            .project_create_vpc(opctx, &authz_project, vpc.clone())
```

Comment:
Doesn't look like this clone is needed, but I could be missing a reference to `vpc` below.

Reply:
Good catch, I wonder why clippy didn't complain? Anyway, fixed in f7ba1eb7.